File: fizzBuzz.swift
func fizzBuzz(n: Int) {
    for i in 1…n{
    if (i%3 != 0 && i%5 != 0){
    print(i)
    }else if (i%3 == 0){
    print(“Fizz”)
    }else if (i%5 == 0){
    print(“Buzz”)
    }else if (i%3 == 0 && i%5 == 0){
    print(“FizzBuzz”)
    }
    }
}

File: fizzbuzz.py
def FizzBuzz():
  for i in range(1,100):
    if i%3==0 and i%5==0:
      print('FizzBuzz', end =",")
      continue;
    if i%3==0:
      print('Fizz', end =",")
      continue;
    if i%5==0:
      print('Buzz', end =",")
      continue;
    else:
      print(i, end =",")
      continue;

FizzBuzz()
